yarn-project/circuits.js/src/contract/contract_class.ts

```diff
@@ -13,7 +13,7 @@ type ContractArtifactWithHash = ContractArtifact & { artifactHash: Fr };
 export function getContractClassFromArtifact(
   artifact: ContractArtifact | ContractArtifactWithHash,
 ): ContractClassWithId {
-  const artifactHash = (artifact as ContractArtifactWithHash).artifactHash ?? getArtifactHash(artifact);
+  const artifactHash = (artifact as ContractArtifactWithHash).artifactHash ?? computeArtifactHash(artifact);
```

Comment:
I asked chatgpt if there was a cleaner way to do this, and FWIW:
```ts
const artifactHash = 'artifactHash' in artifact
    ? artifact.artifactHash
    : computeArtifactHash(artifact);
```

Reply:
Nice, thanks! I'll switch to this style.